config/config.kinesis.reference.hocon

```diff
@@ -202,4 +204,34 @@
       "cloudwatch": false
     }
   }
+
+  # Optional, configure telemetry
+  # All the fields are optional
+  telemetry {
+    # Set to true to disable telemetry
+    disable = false
+    # Interval for the heartbeat event
+    interval = 15 minutes
+    # HTTP method used to send the heartbeat event
+    method = POST
+    # URI of the collector receiving the heartbeat event
+    url = collector-g.snowplowanalytics.com
```

Comment:
Also not consistent how we configure collector in other places (`collectorUri` is used I think?)